demo/src/pages/interApplicationBusSubscribe/InterApplicationBusSubscribe.tsx

```diff
@@ -50,7 +53,7 @@ const InterApplicationBusSubscribe: React.FC = () => {
             <div className={styles.containerMessage}>
                 <div>{isSubscribed ? "Subscribed" : "Subscribing"} to <strong>demo-topic</strong></div>
                 <br/>
-                <div>Last received message: {data ? data.message : "None"}</div>
+                <div>Last received message: {numMessages ? numMessages : "None"}</div>
```

Comment:
is "Last received message" still the right label?

Reply:
It still represents the same things, both the last received message, and the number of messages (which is the content of the message). Let me know if you come up with a better phrase!